commit 7da6b2e062621c423a56a8c0385888557ada6d1a (refs/changes/29/5629/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-02-25T22:33:29+00:00 (8 months ago)
    
    OS-1142 want open source bnx driver
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/overlay/generic/kernel/drv/amd64/bnx b/overlay/generic/kernel/drv/amd64/bnx
deleted file mode 100755
index 24b0db56..00000000
Binary files a/overlay/generic/kernel/drv/amd64/bnx and /dev/null differ
